content/docs/languages/go/basics.md

```diff
@@ -207,19 +208,20 @@ func (s *routeGuideServer) RouteChat(stream pb.RouteGuide_RouteChatServer) error
 ```
 
 ##### Simple RPC
-`routeGuideServer` implements all our service methods. Let's look at the
+
+The `routeGuideServer` implements all our service methods. Let's look at the
 simplest type first, `GetFeature`, which just gets a `Point` from the client and
 returns the corresponding feature information from its database in a `Feature`.
 
 ```go
 func (s *routeGuideServer) GetFeature(ctx context.Context, point *pb.Point) (*pb.Feature, error) {
-	for _, feature := range s.savedFeatures {
-		if proto.Equal(feature.Location, point) {
-			return feature, nil
-		}
-	}
-	// No feature was found, return an unnamed feature
-	return &pb.Feature{"", point}, nil
+  for _, feature := range s.savedFeatures {
```

Comment:
Why untabify?

Reply:
It is my experience that embedding tabs in code excerpts (in a markdown file) can lead to problems in terms of how the code finally gets displayed -- not all code prettifiers treat tabs in the same way IIRC.